### ISSUE TITLE
Controls addon UX

### ISSUE THREAD

**Issue by Hurtak** [tajo/ladle#288]
**Is your feature request related to a problem? Please describe.**
This will probably vary between apps and how people are used to use storybook but from what I have seen the add-ons usage was like this
- controls: they were in almost every story and used heavily
- dark theme and viewport: very useful but used only once in a while, probably order of magnitude less usage than controls
- other add-ons: used even less, probably another order of magnitude less compared to dark theme and viewport

If this usage patterns are correct and not just some isolated outliers (or just me having wrong idea how stories are used) I think it could make sense, from UX point of view, to give the controls add-on more space/priority in the UI, compared to other add-ons.

**Describe the solution you'd like**
For example storybook has dedicated panel in layout for controls that is always visible unless you minimalize it, so we could do something along these lines?

**Describe alternatives you've considered**
Or we could just have some simple adjustments like "if there are controls for given story, open by default the controls add-on panel

TLDR: If most stories use controls, it is little annoying that you need to re-open controls panel whenever you switch between stories

**Comment by tajo** [comment]
Yea, controls are probably the most used addon. 

> Or we could just have some simple adjustments like "if there are controls for given story, open by default the controls add-on panel

This could be an opt-in part of config for this addon. Maybe even with additional options:

- closed by default
- open by default
- open if control values are changed/active

Might make sense to also consider some redesign. I didn't put much effort into the styling of controls initially.
